.github/workflows/redis-rs.yml

```diff
@@ -0,0 +1,124 @@
+name: Redis-rs CI
+
+on:
```

Comment:
All our CI use idents with 4 spaces. Consider updating this.

Reply:
Do we have a consistent tool we use?

Reply:
No, but I found that prettier can do this for us.